README.md

```diff
@@ -452,7 +452,8 @@ services first.
 This starter kit is compatible with Docker for Windows, so you can enjoy native Docker experience on Windows. You will have to keep in mind some differences:
 
 - Composer cache can't be set to the relative home path in `infrastructure/docker/docker-compose.builder.yml`: remove `- "~/.composer/cache:/home/app/.composer/cache"`;
-- You will be prompted to run the env vars manually if you use PowerShell.
+- You will be prompted to run the env vars manually if you use PowerShell;
+- As pty in invoke does not works on Windows, the builder is not really usable...
```

Comment:
I fear this is a block. How can one work on a project without a proper terminal session :(

Reply:
Pending issue about this: https://github.com/pyinvoke/invoke/issues/561